sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -162,10 +166,10 @@ foreach (FormPage page in formPages)
 ### Recognize Custom Forms
 Recognize and extract form fields and other content from your custom forms, using models you train with your own form types.
 
-```C# Snippet:FormRecognizerSample3RecognizeCustomFormsFromUri
+```C# Snippet:FormRecognizerSampleRecognizeCustomFormsFromUri
```

Comment:
Removing the number from snippets labels (`Sample3` => `Sample`). We've reordered them a bit, and most are not consistent anymore. We're likely to do the same in the future, so it would be a bit bothersome to keep the numbers up to date.